modules/local/simpleaf_index.nf

```diff
@@ -2,10 +2,10 @@ process SIMPLEAF_INDEX {
     tag "$transcript_gtf"
     label "process_medium"
 
-    conda 'bioconda::simpleaf=0.10.0-1'
```

Comment:
Would it be possible to switch to the central `nf-core/modules` version at this point (see also https://github.com/nf-core/scrnaseq/issues/296)?

There are modules for simpleaf_index and simpleaf_quant already, they might also need slight updates though.

Reply:
yes please! Once the module PRs are merged, you can simply install them with the `nf-core tools` CLI that you also use for linting: 
```
nf-core modules install
```

Reply:
Cool! Thanks for the information. I will work on this next week! BTW, the other account, an-altosian, is also me :)